fluidkeys/main.go

```diff
@@ -231,14 +231,22 @@ func generatePgpKey(email string, channel chan generatePgpKeyResult) {
 
 func listKeysForImportingFromGpg(secretKeyListings []gpgwrapper.SecretKeyListing) string {
 	str := fmt.Sprintf("Found %s in GnuPG:\n\n", humanize.Plural(len(secretKeyListings), "key", "keys"))
-	for _, key := range secretKeyListings {
-		str += printSecretKeyListing(key)
+	for index, key := range secretKeyListings {
+		str += printSecretKeyListing(index+1, key)
 	}
 	return str
 }
 
-func printSecretKeyListing(key gpgwrapper.SecretKeyListing) string {
-	output := fmt.Sprintf("    %s\n", key.Fingerprint)
+func printSecretKeyListing(listNumber int, key gpgwrapper.SecretKeyListing) string {
+	runeLengthListNumber := len(strconv.Itoa(listNumber) + ".")
```

Comment:
I think you might be able to replace all this code with a fixed-width format specifier like this:
```
fmt.Printf("%-5sblah\n", "1.")
fmt.Printf("%-5sblah\n", "99.")
```
The `-5` means "5-characters wide, left aligned"
https://play.golang.org/p/CbXqrbuytmQ